src/angular/styles/_typography.scss

```diff
@@ -1416,6 +1416,17 @@ textarea {
   height: calc(#{sbb.pxToRem(36)} * var(--sbb-scaling-factor));
   min-width: auto;
   line-height: 0 !important;
+
+  &:is(:disabled, .sbb-disabled):before {
+    content: '';
+    position: absolute;
+    top: 50%;
+    height: 1px;
+    left: calc(#{sbb.pxToRem(5)} * var(--sbb-scaling-factor));
```

Comment:
the 5 pixels look a little bit magic to me (maybe just add a comment) and seem to be wrong too (comparing to Figma). I think you can define the width to var(--sbb-icon-size-default) and center it absolutely in x and y (looks in Figma as it is designed like that). But you can also argue that if someone changes the icon size, the line should adapt itself.